lib/elixir/lib/map.ex

```diff
@@ -515,6 +515,10 @@ defmodule Map do
       nil
       iex> Map.get(%{a: 1}, :b, 3)
       3
+      iex> Map.get(%{a: nil}, :a, 1)
+      nil
+      iex> Map.get(%{a: nil}, :a) || 1
+      1
```

Comment:
I’m not sure this is an example that belongs in the documentation for `Map.get/3`? 🤔

Reply:
Happy to drop it.